lib/ansible/cli/doc.py

```diff
@@ -312,10 +312,10 @@ class DocCLI(CLI, RoleMixin):
     _RULER = re.compile(r"\bHORIZONTALLINE\b")
 
     # rst specific
-    _REFTAG = re.compile(r":ref:")
-    _TERM = re.compile(r":term:")
-    _NOTES = re.compile(r".. note:")
-    _SEEALSO = re.compile(r"^\s*.. seealso:.*$", re.MULTILINE)
+    _RST_ANCHOR = re.compile(r"^\s*.. \w+?::.*$", re.MULTILINE)
+    _RST_REFTAG = re.compile(r":\w+?:`")
+    _RST_NOTES = re.compile(r".. note::")  # subset of _ANCHOR
```

Comment:
Maybe also do
```suggestion
    _RST_NOTE = re.compile(r".. note::")  # subset of _ANCHOR
```
to avoid future label vs content confusion here?